src/gerbil/runtime/init.ss

```diff
@@ -238,6 +238,10 @@ namespace: #f
      (else
       (gx#core-eval-module obj)))))
 
+(def (gerbil-path)
+  (or (getenv "GERBIL_PATH" #f)
+      (path-expand "~/.gerbil")))
```

Comment:
```
(path-expand
  (getenv "GERBIL_PATH" "~/.gerbil"))
```

Reply:
so that the user variable can contain tildes.

Reply:
What does it even mean for the user variable to contain tilde? This is very non-Unixy.

Reply:
It can be set programmatically.